doc/contributing/coding-conventions.xml

```diff
@@ -178,6 +178,12 @@ args.stdenv.mkDerivation (args // {
 </programlisting>
     </para>
    </listitem>
+   <listitem>
+    <para>
+     Arguments should be listed in the order they are used, with the
```

Comment:
Maybe this should say “Formal arguments”?

Reply:
Perhaps even "Formal parameters".